### ISSUE TITLE
Authorship highlighting is not accessible

### ISSUE THREAD

**Issue by bwreid** [WikiEducationFoundation/WikiEduDashboard#2522]
![image](https://user-images.githubusercontent.com/1316902/53369922-2fba4d80-3901-11e9-8de3-ac124d505d71.png)

When reading the diff viewer with a screen reader, it is not possible to differentiate between what text was already on the Wikipedia article and what text was added by a user. Ideally, something would cue an individual reading a screen reader that they were starting to read an edit by a user.
